plugins/contentstack.js

```diff
@@ -7,6 +7,9 @@ const Stack = contentstack.Stack({
   api_key: process.env.CONTENTSTACK_API_KEY,
   delivery_token: process.env.CONTENTSTACK_DELIVERY_TOKEN,
   environment: process.env.CONTENTSTACK_ENVIRONMENT,
+  branch: process.env.CONTENTSTACK_BRANCH
+    ? process.env.CONTENTSTACK_BRANCH
+    : 'main',
   region: process.env.CONTENTSTACK_REGION
```

Comment:
this was already present?

Reply:
Hi @surajcontentstack , region support was already present, I have added the branch support, few of the starters already have region support(I have the list)